src/HarID/Provider.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace SocialiteProviders\HarID;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'HARID';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'openid',
+        'profile',
+        'email',
+        'session_type',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function additionalConfigKeys()
+    {
+        return ['use_test_idp'];
+    }
+
+    /**
+     * Returns base URL for idP endpoints with check for test or live environment.
+     *
+     * @return string
+     */
+    protected function getEndpointBaseUrl()
+    {
+        return ($this->getConfig('use_test_idp', false) === true) ? 'https://test.harid.ee/et' : 'https://harid.ee/et';
```

Comment:
```suggestion
        return (bool) $this->getConfig('use_test_idp', false) === true ? 'https://test.harid.ee/et' : 'https://harid.ee/et';
```

Reply:
I've had a similar code before and changed that to be strict check as almost any value passed to the `use_test_idp` config setting would make it true. I'm not against the change though as it does not really matter all that much.